tests/integration/route_test.py

```diff
@@ -1884,3 +1884,44 @@ def test_route_rule_suppress_prefix_length(route_rule_test_env):
     }
     libnmstate.apply(desired_state)
     _check_ip_rules(desired_state[RouteRule.KEY][RouteRule.CONFIG])
+
+
+def test_append_route_rule(route_rule_test_env):
```

Comment:
Can we mark it as tier 1 ?

Reply:
Considering this issue has been there for a while and we never got complain from our consumers, I would assume the do not have this needs yet.

Let's leave it as tier2 and promote it after we have a link to refer.